app/src/common/constants/pluginsGroupTypes.js

```diff
@@ -21,3 +21,13 @@ export const BTS_GROUP_TYPE = 'BTS';
 export const ANALYZER_GROUP_TYPE = 'ANALYZER';
 export const IMPORT_GROUP_TYPE = 'IMPORT';
 export const OTHER_GROUP_TYPE = 'OTHER';
+
+export const PLUGIN_LABELS = {
+  [ALL_GROUP_TYPE]: 'All',
+  [NOTIFICATION_GROUP_TYPE]: 'Notifications',
+  [AUTHORIZATION_GROUP_TYPE]: 'Authorization',
+  [BTS_GROUP_TYPE]: 'Bug Tracking Systems',
+  [ANALYZER_GROUP_TYPE]: 'Analyzer',
+  [IMPORT_GROUP_TYPE]: 'Launches Import',
+  [OTHER_GROUP_TYPE]: 'Other',
+};
```

Comment:
Can we use the group types as is for event labels to avoid creating extra mappings and texts?)
Please discuss with WA.
In case it's crucial to have more detailed labels, let's move them to the corresponding analytics folder.